articles/quickstart/backend/rails/interactive.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Add authorization to a Ruby on Rails API
+description: This tutorial demonstrates how to add authorization to a Ruby on Rails API.
+topics:
+    - quickstart
+    - backend
+    - rails
+github:
+  path: 01-Authentication-RS256
+contentType: tutorial
+useCase: quickstart
+interactive: true
+files:
+  - files/json_web_token
+  - files/secured
+  - files/public
+  - files/private
+---
+<!-- markdownlint-disable MD041 MD025 -->
+
+# Add authorization to a Ruby on Rails API
+This tutorial performs Access Token validation using the  [jwt](https://github.com/jwt/ruby-jwt) Gem. A Concern called `Secured` is used to mark endpoints which require authentication through an incoming Access Token.
+
+If you haven't created an API in your Auth0 dashboard yet, you can use the interactive selector to create a new Auth0 API or select an existing API that represents the project you want to integrate with.
+
+Alternatively, you can read [our getting started guide](get-started/auth0-overview/set-up-apis) that helps you set up your first API through the Auth0 dashboard.
+
+Every API in Auth0 is configured using an API Identifier that your application code will use as the Audience to validate the access token.
+
+<%= include('../../../_includes/_api_auth_intro') %>
+
+## Define permissions
+<%= include('../_includes/_api_scopes_access_resources') %>
+
+## Install dependencies
+Install the **jwt** Gem.
+
+```bash
+gem 'jwt'
+bundle install
+```
+
+## Create a JsonWebToken class {{{ data-action=code data-code="lib/json_web_token.rb" }}}
+
+Create a class called `JsonWebToken` which decodes and verifies the incoming Access Token taken from the `Authorization` header of the request.
+
+This will fetch the public key for your Auth0 tenant then use it to verify the signature of the Access Token.
```

Comment:
Style guide (word list): access token should be lowercase

```suggestion
This will fetch the public key for your Auth0 tenant then use it to verify the signature of the access token.
```